docs/quickstart.rst

```diff
@@ -279,48 +258,42 @@ Each SyncroSim Library contains multiple SyncroSim `Datasheets`_. A SyncroSim Da
 
     # View a summary of all Datasheets associated with the Scenario
     >>> my_scenario.datasheets()
-                Package                              Name                Display Name
-    0              core              core_AutoGenTagValue  Auto Generation Tag Values
-    1              core                     core_Pipeline                    Pipeline
-    2         corestime       corestime_DistributionValue               Distributions
-    3         corestime                corestime_External                    External
-    4         corestime   corestime_ExternalVariableValue          External Variables
-    5         corestime         corestime_Multiprocessing     Spatial Multiprocessing
-    6  helloworldTimePy   helloworldTimePy_InputDatasheet              InputDatasheet
-    7  helloworldTimePy  helloworldTimePy_OutputDatasheet             OutputDatasheet
-    8  helloworldTimePy       helloworldTimePy_RunControl                 Run Control
-    
-Above, we can see that some Datasheets belong to the ``core`` or ``corestime`` Packages. These are Datasheets that come bundled with the SyncroSim software and are not specific to the ``helloworldTimePy`` Package.
+                Package                              Name              DisplayName
+    0              core            core_DistributionValue            Distributions
+    1              core        core_ExternalVariableValue       External Variables
+    2              core                     core_Pipeline                 Pipeline
+    3              core       core_SpatialMultiprocessing  Spatial Multiprocessing
+    4  helloworldTimePy   helloworldTimePy_InputDatasheet                   Inputs
+    5  helloworldTimePy  helloworldTimePy_OutputDatasheet          OutputDatasheet
+    6  helloworldTimePy       helloworldTimePy_RunControl              Run Control
+    
+Above, we can see that some Datasheets belong to the ``core`` Packages. These are Datasheets that come bundled with the SyncroSim software and are not specific to the ``helloworldTimePy`` Package.
```

Comment:
We had this old naming convention where we would capitalize SyncroSim terms like "Datasheets", "Scenarios", "Projects", and "Libraries", but we've since decided that we don't want to do this. Can you go through and uncapitalize these where it makes sense?

Reply:
Are Session, and DataFrame included in this new naming convention?

Reply:
Yes, except for DataFrame (that's an R term)